schema/lrs.fbs

```diff
@@ -99,72 +69,43 @@ table Anchor {
     properties:[Property];
 
     /// Most anchors have a name, which is used to reference the location.
-    /// More often than not a kilometer or mile number, but it can also be a letter or word.
+    /// It can often be a kilometer or mile number, but it can also be a letter or word.
     name:string;
     /// Anchors can be bound to a node, or defined independently.
-    /// If the anchor is bound to a node, it's location is deduced from location of the node.
-    node:NodeRef;
+    /// If the anchor is bound to a node, its location is deduced from location of the node.
+    node:uint32;
+    /// The anchor can also be defined by a geographical position.
+    geometry: Point;
 }
 
 enum DistanceUnit : byte { Meters, MilliMeters }
 
-/// Linear referencing methods (LRMs) are curves in space, along which distances can be measured.
+/// Linear Referencing Methods (LRMs) are curves in space, along which distances can be measured.
 /// Each linear referencing method has:
 ///   * a network traversal, which defines the path of the curve
 ///   * a sequence of anchors, which are projected on the curve, and used as positioning reference points
 ///   * distances between anchors have to be defined: even though distances can be measured on the curve,
-//      it can be imprecise enough to joepardize correct ordering of object positionned relative to different reference points
+//      it can be imprecise enough to joepardize correct ordering of object positioned relative to different reference points.
 table LinearReferencingMethod {
     id:string (required);
     properties:[Property];
 
-    traversal_index:TraversalRef;
-    /// An LRM can apply to multiple traversal
-    /// For instance a LRM can be the central line of a highway
-    /// And that LRM is the reference for the two other traversals corresponding to each direction
-    used_on:[TraversalRef];
+    traversal_index:uint32;
+    /// An LRM can apply to multiple traversal.
+    /// For instance, a LRM can be the central line of a highway.
+    /// And that LRM is the reference for the two other traversals corresponding to each direction.
+    used_on:[uint32];
     anchor_indices:[uint64] (required);
     distances:[double] (required);
-    /// The unit used to measure the distance between anchors
+    /// The unit used to measure the distance between anchorss
```

Comment:
Shouldn't it be just one s? 'anchors.' ?

Reply:
good catch, it’s fixed. Thank you